### PR TITLE
Tdykes.fix.group.refresh

### LINKED ISSUE #114: Pull-to-refresh causes a crash on group detail page
Repro on iOS - unsure if Android affected:
Go to groups tab. Tap a group (tested with Wollongong group). Pull to refresh results in app crash and I get kicked back to springboard.

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
Fixes #114

bind error on function

### CODE REVIEW

File: client/src/screens/group.screen.js [hunk 1]
```diff
@@ -206,8 +207,6 @@ class Group extends Component {
 
   leaveGroup() {
     this.props.leaveGroupQry({ groupId: this.getGroup().id }).then(() => {
-      // pull group details down again to populate the group details
-      this.onRefresh();
```
Comment (sdunster) [inline client/src/screens/group.screen.js]: this seems like an unrelated change?

Comment (OSPFNeighbour) [reply client/src/screens/group.screen.js]: It's semi related as it's a totally useless refresh that was working while the other failed.

Comment (sdunster) [reply client/src/screens/group.screen.js]: 👍
